packages/side-nav/src/CollapsedSideNavItem/CollapsedSideNavItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import Portal from '@leafygreen-ui/portal';
+
+import { useSideNavContext } from '../SideNav/SideNavContext';
+
+import { activeThemeStyle, baseStyles, themeStyle } from './styles';
+import { CollapsedSideNavItemProps } from './types';
+
+export function CollapsedSideNavItem({
+  children,
+  active = false,
+  className,
+}: CollapsedSideNavItemProps) {
+  const { portalContainer, theme } = useSideNavContext();
+
+  return (
+    <Portal container={portalContainer}>
+      <li
+        className={cx(
+          baseStyles,
+          themeStyle[theme],
+          {
+            [activeThemeStyle[theme]]: active,
+          },
+          className,
+        )}
+      >
+        {children}
+      </li>
+    </Portal>
+  );
+}
```

Comment:
Thoughts on moving the `Portal` to its parents component and extending `HTMLElementProps<HTMLListItemElement>` in the types?

Reply:
This is an external component that consumers can use so we have to keep the portal inside but I think we can still extend it and spread a rest on the li, thoughts?

Reply:
Actually i'm going to leave as is for now. This component is used inside `SideNavItem` and consumers can also use it as a direct child of `SideNav` so the types would also need to change for `SideNavItem`. We should change the types in a separate PR.